src/m4a.c

```diff
@@ -1198,8 +1198,8 @@ void CgbSound(void)
             }
             else
             {
-                envelopeStepTimeAndDir &= 0xf;
-                *nrx2ptr = (channels->envelopeVolume << 4) + envelopeStepTimeAndDir;
+                unsigned int envMask = 0xF;
```

Comment:
`u32` (or `u8`, it doesn't really matter here). In any case I doubt `envMask` is needed, why not just use `0xF` directly

Reply:
It's required on older m4a versions and with this var it matches both ways, so I think it was like this originally and direct use is actually a fakematch.

Reply:
It's not needed here, but it is needed in previous revisions, according to Revo, so I kept it like that for consistency. I asked the same thing. I can change it to u32 though, that's fair